pennylane/math/single_dispatch.py

```diff
@@ -400,6 +400,9 @@ def _cond_tf(pred, true_fn, false_fn, args):
 ar.register_function("tensorflow", "cond", _cond_tf)
 
 
+ar.register_function("tensorflow", "vander", _i("tf").experimental.numpy.vander)

```

Comment:
On this branch, importing PennyLane leads to the following message. Is this line the cause for that? I am bit puzzled by this behavior (I checked and I dont get it on master).

```
2022-07-02 12:33:17.060342: I tensorflow/core/util/util.cc:169] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2022-07-02 12:33:17.067311: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2022-07-02 12:33:17.067360: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
```

Reply:
I also just noticed that this line is causing problems in the tests overall

Reply:
Hm strange, I have the same tensorflow and tensorflow-estimator versions.
On the CI tests here, the `core-and-interface-tests (3.7, core)` for example breaks with
```pycon
Run python -m pytest tests --cov=pennylane $COVERAGE_FLAGS -m "$SUITE" -n auto
ImportError while loading conftest '/home/runner/work/pennylane/pennylane/tests/conftest.py'.
tests/conftest.py:23: in <module>
    import pennylane as qml
(...)
 pennylane/math/single_dispatch.py:403: in <module>
    ar.register_function("tensorflow", "vander", _i("tf").experimental.numpy.vander)
pennylane/math/single_dispatch.py:28: in _i
    return import_module("tensorflow")
E   ModuleNotFoundError: No module named 'tensorflow'
Error: Process completed with exit code 4.
```

Reply:
resolved